<p>File an issue report on <a href="https://github.com/bmabey/pyLDAvis/issues" rel="nofollow">pyLDAvis's GitHub</a>. It looks like a <a href="https://github.com/bmabey/pyLDAvis/commit/848b247b77bb24b1e674aa0cf7c0d5f9b4f26a9d" rel="nofollow">recent change</a> broke Python 2 compatibility by assuming Python 3 absolute import behavior for <code>import gensim</code>.</p>

<p>In the meantime, I believe the bug isn't present in the 1.3.1 release, so you could use that. Alternatively, you could edit <code>pyLDAvis/gensim.py</code> and add <code>from __future__ import absolute_import</code> at the top. That'd <em>probably</em> work as a temporary fix, but I didn't try it.</p>
